api/api-rules/violation_exceptions.list

```diff
@@ -257,7 +257,12 @@ API rule violation: list_type_missing,k8s.io/api/extensions/v1beta1,RunAsUserStr
 API rule violation: list_type_missing,k8s.io/api/extensions/v1beta1,RuntimeClassStrategyOptions,AllowedRuntimeClassNames
 API rule violation: list_type_missing,k8s.io/api/extensions/v1beta1,SupplementalGroupsStrategyOptions,Ranges
 API rule violation: list_type_missing,k8s.io/api/imagepolicy/v1alpha1,ImageReviewSpec,Containers
+API rule violation: list_type_missing,k8s.io/api/networking/v1,HTTPIngressRuleValue,Paths
 API rule violation: list_type_missing,k8s.io/api/networking/v1,IPBlock,Except
+API rule violation: list_type_missing,k8s.io/api/networking/v1,IngressList,Items
```

Comment:
Rebasing/regenerating on master will drop the IngressList Items violation. I'd expect us to actually address the others by defining listType rather than expanding exceptions. cc @apelisse @lavalamp on recommended listTypes for these